v2/manifests/core-ons/zebedee.yml

```diff
@@ -49,10 +49,13 @@ services:
       website_reindex_key:            "1hZiEDeZcVKZwO6WmTDTDhVSiRAKS0jM6Nzlvlszk0OW0vY5M2FCiGD7ncqcucxB"
       scheduled_publishing_enabled:   "false"
       OTEL_JAVAAGENT_ENABLED:         ${OTEL_JAVAAGENT_ENABLED:-"false"}
-      ELASTIC_SEARCH_CLUSTER:         "elasticsearch"
-      ELASTIC_SEARCH_SERVER:          "elasticsearch"
+      ELASTIC_SEARCH_CLUSTER:         "cmdelasticsearch"
+      ELASTIC_SEARCH_SERVER:          "cmdelasticsearch"
```

Comment:
This is setting this as the ES server for all consumers - think this should be overridden at the stack level.

Reply:
This has now been moved to the cmd stack file